src/main.cpp

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright (C) 2020 YADRO
+// Copyright (C) 2021 YADRO
```

Comment:
It is a bad practice to change the date in copyrights.

https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated:
The copyright notice on a work establishes a claim to copyright. The date on the notice establishes how far back the claim is made. This means if you update the date, you are no longer claiming the copyright for the original date and that means if somebody has copied the work in the meantime and they claim its theirs on the ground that their publishing the copy was before your claim, then it will be difficult to establish who is the originator of the work.

Reply:
reverted